client/core/core.go

```diff
@@ -4293,8 +4294,11 @@ func (c *Core) Login(pw []byte) error {
 		// resolveActiveTrades. We won't try to unlock here, but if the wallet
 		// is needed for active trades, it will be unlocked in resolveActiveTrades
 		// and the balance updated there.
+		c.notify(newLoginNote("Connecting wallets..."))
 		c.connectWallets()
+		c.notify(newLoginNote("Resolving active trades..."))
```

Comment:
"Resolving" is unclear in meaning.  For the user, I think we should just say "Loading" or "Preparing" or "Resuming".  I'm leaning toward "Resuming".